core/domain/values/session/SessionLifespan.php

```diff
@@ -104,11 +104,11 @@ public function inHoursMinutesSeconds($separator = ':')
      *
      * @param bool $utc If true, displays expiration in UTC
      *                  If false, displays expiration in local time
-     * @return int
+     * @return float
      */
     public function expiration($utc = true)
     {
-        return (int) current_time('timestamp', $utc) - $this->lifespan;
+        return current_time('timestamp', $utc) - $this->lifespan;
```

Comment:
did you merge from 11400 or directly from master?  This looks like a revert of changes made on master.  As mentioned yesterday I already took care of a bunch of conflicts that happened when merging master into 11400 so if you merged from 11400 then I might have resolved this conflict wrong. Yet, I'm pretty sure I resolved it to use `int` (from master) so the fact that I noticed this  here is kind of odd.

Reply:
branch just needed updating. is correct now